core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -34,12 +36,53 @@ abstract class SparkFunSuite
   with Logging {
 // scalastyle:on
 
+  val threadWhiteList = Set(
+    /**
+     * Netty related threads.
+     */
+    "netty.*",
+
+    /**
+     * A Single-thread singleton EventExecutor inside netty which creates such threads.
+     */
+    "globalEventExecutor.*",
+
+    /**
+     * Netty creates such threads.
```

Comment:
But are these stoppable or just out of our control? That's the kind of thing that should define whether something is in the whitelist or not.

Reply:
This is netty internal what we're not configuring or using directly.

Reply:
Note I want these explanations in the code comments, not here in the PR.

Reply:
Ah, OK :)

Reply:
Explanation added.